### DISCUSSION TITLE
ManualContent.php | Fatal error: Uncaught Error: Call to undefined function _()

### DISCUSSION THREAD

**Discussion by irik77587** [timschofield/webERP#169] [Problems / Bugs] [answered]
After installing weberp into laragon and completing setup I clicked the Manual button to find this:

Fatal error: Uncaught Error: Call to undefined function _() in path\to\weberp\ManualContents.php:21 Stack trace: #0 {main} thrown in path\to\weberp\ManualContents.php on line 21

Then I opened the ManualContents.php file in editor and changed the lines from this:

```
$PageSecurity = 0;
$Title = _('webERP Manual');
// Set the language to show the manual:
/*
session_start();
$Language = $_SESSION['Language'];
if(isset($_GET['Language'])) {// Set an other language for manual.
	$Language = $_GET['Language'];
}
*/
include('includes/session.php');
```
to this:
```

$PageSecurity = 0;

include('includes/session.php');
$Title = _('webERP Manual');
// Set the language to show the manual:
/*
session_start();
$Language = $_SESSION['Language'];
if(isset($_GET['Language'])) {// Set an other language for manual.
	$Language = $_GET['Language'];
}
*/

```
Now when I click on Manual from dashboard, the page appears properly

**Comment by timschofield** [comment, marked as answer]
I suspect that you don't have the php-gettext extension installed which is why nobody else has reported this. If this extension is not installed webERP tries to use a different system, but this is only included inside session.php. You should probably get the extension installed if it is not.

That said your fix is good in any case, so I would raise a pull request for it.

Tim
